neutron/neutron.go

```diff
@@ -231,6 +233,85 @@ func (c *Client) DeleteFloatingIPV2(ipId string) error {
 	return err
 }
 
+// PortV2 describes a defined network for creating a port.
+type PortV2 struct {
+	Description    string           `json:"description,omitempty"`
+	FixedIPs       []PortFixedIPsV2 `json:"fixed_ips,omitempty"`
+	Id             string           `json:"id,omitempty"`
+	Name           string           `json:"name,omitempty"`
+	NetworkId      string           `json:"network_id"`
+	SecurityGroups []string         `json:"security_groups,omitempty"`
+	Status         string           `json:"status,omitempty"`
+	TenantId       string           `json:"tenant_id,omitempty"`
```

Comment:
"port_security_enabled" might be useful too.  I know we need it for the networks, not sure if it can be port specific or not.

"device_id" might be useful too, for mapping to server instances.